bin/metababel

```diff
@@ -23,6 +23,31 @@ def sanitize(str)
   str.gsub(/[^0-9A-Za-z-]/, '_')
 end
 
+def btx_match?(str, pattern_or_str)
+  # String beginning and ending in '/' are treated as regexp.
+  match = pattern_or_str.match(%r{^/(?<expr>.+)/$})
+  match ? str.match?(match['expr']) : str == pattern_or_str
```

Comment:
Is this a little more readable?
```
is_regex = pattern.match(%r{^/.+/$})
is_regex ? str =~ eval(pattern) : str == pattern
```
It feed weird to me to extract the thing between  `//` and the using it. Took me will to understand each time

Reply:
This looks nice !!. On the other hand, I am wondering if we can inyect ruby code into the pattern, but I didn't find how, so maybe this is safe for now.

Reply:
if they put `rm -rf *` in the name indeed bad thing will happened... But honestly we don't care